internal/controllers/machinedeployment/testdata/rollingupdate/in-place rollout, 12 replicas, maxsurge 3, maxunavailable 1, scale down to 6, random(0).test.log.golden

```diff
@@ -0,0 +1,95 @@
+  - ms1, 2/2 replicas (m11,m12 => ms2)
+  - ms2, 3/6 replicas (m1,m2,m3,m4🟡,m5🟡,m6🟡 <= ms1)
+[MS controller] Iteration 3, Reconcile ms2, 3/6 replicas (m1,m2,m3,m4🟡,m5🟡,m6🟡 <= ms1)
+[MD controller] Iteration 4, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 5/6 replicas
+  - ms1, 2/2 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4🟡,m5🟡,m6🟡 <= ms1)
+[MD controller] - Result of rollout planner
+  md, 8/6 replicas
+  - ms1, 2/2 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4🟡,m5🟡,m6🟡 <= ms1)
+[MS controller] Iteration 4, Reconcile ms1, 2/2 replicas (m11,m12 => ms2)
+[MS controller] Iteration 4, Reconcile ms2, 6/6 replicas (m1,m2,m3,m4🟡,m5🟡,m6🟡 <= ms1)
+[MS controller] - Replicas m4,m5,m6 completed in place update
+[MS controller] Iteration 5, Reconcile ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] Iteration 5, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 8/6 replicas
+  - ms1, 2/2 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] - Result of rollout planner
+  md, 8/6 replicas
+  - ms1, 2/0 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MS controller] Iteration 6, Reconcile ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] Iteration 7, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 8/6 replicas
+  - ms1, 2/0 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] - Result of rollout planner
+  md, 8/6 replicas
+  - ms1, 2/0 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MS controller] Iteration 7, Reconcile ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] Iteration 8, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 8/6 replicas
+  - ms1, 2/0 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] - Result of rollout planner
+  md, 8/6 replicas
+  - ms1, 2/0 replicas (m11,m12 => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MS controller] Iteration 8, Reconcile ms1, 2/0 replicas (m11,m12 => ms2)
+[MS controller] - Move capped to 0 replicas to avoid unnecessary in-place upgrades
+[MS controller] - ms1 scale down to 0/0 replicas ( moved to ms2)
+[MS controller] - ms1 scale down to 0/0 replicas (m11,m12 deleted)
+[MS controller] Iteration 9, Reconcile ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MS controller] Iteration 9, Reconcile ms1, 0/0 replicas ( => ms2)
+[MD controller] Iteration 9, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 8/6 replicas
+  - ms1, 0/0 replicas ( => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[MD controller] - Result of rollout planner
+  md, 6/6 replicas
+  - ms1, 0/0 replicas ( => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
+[Test] Final state
+  md, 6/6 replicas
+  - ms1, 0/0 replicas ( => ms2)
+  - ms2, 6/6 replicas (m1,m2,m3,m4,m5,m6 <= ms1)
```

Comment:
Just one comment about the golden files. Looks like we still have the annotations at the end. Is this intended?